CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+Contributing
+============
+
+If you’re not sure this is the correct place to file an issue then please file an issue on the [mozilla/addons] project instead.
+
+Before contributing code, please note:
+
+-   You agree to license your contributions under the [license].
+-   Please ask on the [dev-addons mailing list] before submitting pull-requests for new features or large changes that are not related to existing issues.
```

Comment:
I would say "Please file issues or ask on the dev-addons list". No reason the discussion can't happen on here.